### PR TITLE
Add Weights & Biases example

### LINKED ISSUE #41: Add `WeightsAndBiasesCallback` example
## Motivation

Optuna introduces W&B integration, [`WeightsAndBiasesCallback`](https://optuna.readthedocs.io/en/latest/reference/generated/optuna.integration.WeightsAndBiasesCallback.html#optuna.integration.WeightsAndBiasesCallback) to track optimisation since the next version update, v2.9.0. By following our convention, it would be great to provide an example code to use the callback functionality. 

## Description

The expected pull request contains 

1. minimal example code like [`mlflow/keras_mlflow.py`](https://github.com/optuna/optuna-examples/blob/main/mlflow/keras_mlflow.py)
2. `README.md` like [`mlflow/README.md`](https://github.com/optuna/optuna-examples/tree/main/mlflow); 
3. Add weights and biases to [`README.md`](https://github.com/optuna/optuna-examples/blob/main/README.md)

Of course, `weightsandbiases/requirement.txt` and `.github/workflows/weightsandbiases.yml` are necessary.

## Additional context (optional)

Add W&B's link from the main repository: 

- https://github.com/optuna/optuna#integrations
- https://github.com/optuna/optuna/tree/master/examples

### PR DESCRIPTION
**Pull request by xadrianzetx**
## Motivation

This PR adds example for `WeightsAndBiasesCallback` introduced in https://github.com/optuna/optuna/pull/2781. Resolves #41.

## Description of the changes

* [x] Add example script
* [x] Add CI job
* [x] Update README

### CODE REVIEW

**Comment by xadrianzetx** [comment]
[Here's](https://github.com/xadrianzetx/optuna-examples/blob/example-wandb/wandb/README.md) how `wandb/README` renders with screenshot.

File: wandb/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Tracking optimization process with Weights & Biases
+
+![wandbui](https://user-images.githubusercontent.com/37713008/133468378-3eae55e3-dd07-4e87-b3cf-f60c06fd061d.png)
+
+Optuna example that optimizes a classifier configuration for the Iris dataset usinng
```
Comment (nzw0301) [inline wandb/README.md:5]:
```suggestion
Optuna example that optimizes a classifier configuration for the Iris dataset using
```

File: wandb/wandb_simple.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+"""
+Optuna example that optimizes a classifier configuration for the Iris dataset usinng
```
Comment (nzw0301) [inline wandb/wandb_simple.py:2]:
```suggestion
Optuna example that optimizes a classifier configuration for the Iris dataset using
```

**Review by nzw0301 — CHANGES_REQUESTED** [review]
LGTM except for minor typos. Could you fix them?

File: wandb/README.md (hunk 1 above)
Comment (xadrianzetx) [reply wandb/README.md]: Fixed in 03805bd1f70073201ac4e53bbeeff82820d26e69.

**Review by nzw0301 — APPROVED** [review]
Thank you for your quick update as always!

**Pull request merged**